usage/test-framework/src/main/java/org/apache/brooklyn/test/framework/SimpleShellCommandTest.java

```diff
@@ -87,21 +90,24 @@
      * If not explicitly configured, the default assertion is a non-zero exit code.
      */
     @SetFromFlag("assertStatus")
-    ConfigKey<Map> ASSERT_STATUS = ConfigKeys.newConfigKey(Map.class, "assert.status",
-            "Assertions on command exit code", Maps.newLinkedHashMap());
+    ConfigKey<List<Map<String, Object>>> ASSERT_STATUS = ConfigKeys.newConfigKey(
```

Comment:
sadly you can't do `ConfigKey<Map | List>` :) so the way is just `ConfigKey<Object>` and use a static method to coerce the result.